src/main/php/lang.base.php

```diff
@@ -13,7 +13,8 @@ final class xp {
     'xp'     => 'xp',
     'string' => "\xfestring",
     'int'    => "\xfeint",
-    'double' => "\xfedouble",
+    'float'  => "\xfefloat",
+    'double' => "\xfefloat",
```

Comment:
Not sure about this alias

Reply:
That's to support `literal(gettype($v))` when `$v` is a float. PHP is inconsistent in this.